rsk-statistics/index.js

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env node
+import { getDeploymentsNumber, getUniqueDeployersNumber } from './dbQueries.js';
+import dbPool from './dbPool.js';
+
+const startDate = '2022-04-01 00:00';
+const endDate = '2022-05-01 00:00';
```

Comment:
Instead of hard coding these as constants, it would be better if these are read in as CLI params.
e.g. `./rsk-statistics/index.js 2022-04-01 2022-05-01` (and then auto append ` 00:00` if unspecified)